docs/run/configure-error-mitigation.mdx

```diff
@@ -265,13 +376,116 @@ from qiskit_ibm_runtime import QiskitRuntimeService, Sampler, Options
+|            | resilience_level         | How much resilience to build against errors.                                                                                                                                                                                            | 1              | Estimator only |
+|            | dynamical_decoupling     | The dynamical decoupling sequence to use.                                                                                                                                                                                               | None           |                |
+|            | seed_estimator           | Seed used to control sampling.                                                                                                                                                                                                          | None           | Estimator only |
+| resilience | measure_noise_mitigation | Whether to enable measurement error mitigation method.                                                                                                                                                                                  | True           | Estimator only |
+| resilience | zne_noise_factors        | An list of real valued noise factors that determine by what amount the circuits' noise is amplified.                                                                                                                                    |                | Estimator only |
+| resilience | zne_extrapolator         | An extrapolation strategy. One or more of ``multi_exponential``,  ``single_exponential``, ``double_exponential``, ``linear``.                                                                                                           |                | Estimator only |
+| resilience | zne_stderr_threshold     | A standard error threshold for accepting the ZNE result of Pauli basis expectation values when using ZNE mitigation.                                                                                                                    |                | Estimator only |
+| resilience | pec_mitigation           | Whether to turn on the PEC error mitigation method.                                                                                                                                                                                     | False          | Estimator only |
+| resilience | pec_max_overhead         | Specify a maximum sampling overhead for the PEC sampling noise model. If None the full learned model will be sampled from.                                                                                                              |                | Estimator only |
```

Comment:
Can we reword so not to end a sentence with the word "from"? ("If None the full learned model will be sampled from.")

Reply:
Maybe just remove the word from, and it would work?